src/scrabbleScore.js

```diff
@@ -24,6 +24,13 @@ let SCRABBLE_TILE_SCORES = {
  */
 
 function scrabbleScore(word) {
+  word = word.toLowerCase();
+  let sum = 0;
+  for (let i = 0; i < word.length; ++i) {
+    sum += SCRABBLE_TILE_SCORES[word.charAt(i)] || 0;
```

Comment:
Hmm, this _shouldn't_ be necessary unless some of the words contain non-Scrabble characters. I dunno that I'd count them as `0`. In a "real" game of Scrabble, it'd be more like playing a fake tile with a non-Scrabble letter on it.

I'd expect an exception of some kind, e.g., `throw new Error(`Invalid Scrabble letter: '${word.chatAt(i)}'`.

It doesn't really matter in the context of the assessment, though. Good on you for noticing!